libnetwork/internal/rootlessnetns/netns_linux.go

```diff
@@ -143,6 +155,36 @@ func (n *Netns) cleanup() error {
 	return multiErr.ErrorOrNil()
 }
 
+func (n *Netns) setupPasta(nsPath string) error {
+	pastaOpts := pasta.SetupOptions{
+		Config:       n.config,
+		Netns:        nsPath,
+		ExtraOptions: []string{"-p", rootlessNetNsConnPidFile},
```

Comment:
doesn't this create the file in the cwd? It must be wrapped in `n.getPath()` to get a proper absolute path.